media-gfx/blender/blender-9999.ebuild

```diff
@@ -381,10 +395,22 @@ src_test() {
 	[ -d "$BLENDER_SYSTEM_SCRIPTS" ] || die "The custom script path is invalid, fix the ebuild!"
 	[ -d "$BLENDER_SYSTEM_DATAFILES" ] || die "The custom datafiles path is invalid, fix the ebuild!"
 
-	cmake_src_test
+	if use cuda; then
+		cuda_add_sandbox -w
+		addwrite "/dev/dri/renderD128"
+		addwrite "/dev/char/"
+	fi
+
+	if use X; then
+		xdg_environment_reset
+		LIBGL_ALWAYS_SOFTWARE=1 \
+		virtx cmake_src_test
```

Comment:
I'm guessing you added this because you had issues with GL initialization?
It is a bit weird to me though as Blender should now always used EGL and that should work in a headless environment without a X or wayland display.

Where was it failing for you?